releasenotes/notes/sparselabelop-class-57ffd57121dadfe5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds :class:`.SparseLabelOp`.
```

Comment:
Maybe this can be expanded a bit? Stating that this will is a class which maps sparse labels to coefficient values which will become the new base class for the other operator types.